tests/security/ima/ima_measurement.pm

```diff
@@ -40,11 +40,15 @@ sub run {
     $self->wait_boot;
     $self->select_serial_terminal;
 
+    # Upload files for reference before test dies
```

Comment:
Please update the comments here for the reason why need upload file before test die.